packages/main/src/GridRow.ts

```diff
@@ -34,17 +27,16 @@ import {
  * <code>import @ui5/webcomponents/dist/GridRow.js";</code>
  *
  * @constructor
- * @extends UI5Element
+ * @extends GridRowBase
  * @public
  */
 @customElement({
 	tag: "ui5-grid-row",
-	renderer: litRender,
-	styles: GridRowCss,
+	styles: [GridRowBase.styles, GridRowCss],
 	template: GridRowTemplate,
-	dependencies: [RadioButton, CheckBox],
+	dependencies: [RadioButton],
```

Comment:
We still need the CheckBox dependency here, no? Having it solely in GridRowBase does not suffice I think, probably similar to the styles?